packages/react/lib/react-helmet-shim.js

```diff
@@ -0,0 +1,15 @@
+const deprecate = require('depd')('@untool/react');
+const reactHelmetAsync = require('react-helmet-async');
+
+deprecate(
+  '[DEP004] react-helmet is deprecated. Please switch to react-helmet-async instead (https://github.com/untool/untool/blob/master/DEPRECATIONS.md#dep004).'
+);
+
+module.exports = reactHelmetAsync;
+module.exports.default = reactHelmetAsync.Helmet;
```

Comment:
This will probably break for apps that are written in commonjs. Is that a use-case that we want to support as well?

Reply:
react-helmet (v5.2.1) is doing this:
```js
exports.__esModule = true;
...
exports.Helmet = HelmetExport;
exports.default = HelmetExport;
```
(https://unpkg.com/react-helmet@5.2.1/lib/Helmet.js)

And
```js
export { HelmetExport as Helmet };
export default HelmetExport;
```
(https://unpkg.com/react-helmet@5.2.1/es/Helmet.js)

So I think we should do the same here:
```js
exports.__esModule = true;
exports.Helmet = reactHelmetAsync.Helmet;
exports.default = reactHelmetAsync.Helmet;
```